src/mapport.cpp

```diff
@@ -268,7 +176,7 @@ static void DispatchMapPort()
 
     assert(g_mapport_thread.joinable());
     assert(!g_mapport_interrupt);
-    // Interrupt a protocol-specific loop in the ThreadUpnp() or in the ThreadPCP()
```

Comment:
I think you can just drop this comment entirely, as there is no-longer a "next protocol" in the loop.

Reply:
i think cleaning up a bit more makes sense here, though let's keep it easy to review refactor-wise

Reply:
Done.